pages/posts/index.tsx

```diff
@@ -20,28 +24,45 @@ export const getStaticProps: GetStaticProps = async () => {
   return { props: { allPosts: allPosts } };
 };
 
-const Posts: React.FC<Props> = ({ allPosts }) => (
-  <PageWrapper page="/posts" withChrome language="en" redirectTo="/publicaciones">
-    <div className="p-8 md:p-16 dark:bg-slate-900">
-      <h2 className="text-3xl xs:text-4xl font-inter dark:text-white">Posts</h2>
-      <p className="mt-3 text-slate-500">
-        The majority of these posts are my replies to emails, text messages, or other
-        questions brought up in various settings. Any names or personal information have
-        of course been removed.
-      </p>
-    </div>
-    {allPosts.length > 0 ? (
-      <section className="sm:p-16 p-8 pt-12 sm:pt-4 space-y-14 md:space-y-8 relative bg-graph-paper dark:bg-slate-900 dark:[background-image:none]">
-        {allPosts.map((post) => (
-          <PostPreview post={post} key={post.en.id} />
-        ))}
-      </section>
-    ) : (
-      <div className="flex justify-center items-center mt-20 dark:bg-slate-900">
-        <i className="fa-solid fa-spinner text-5xl text-sky-500 text-opacity-30 animate-spin" />
+const Posts: React.FC<Props> = ({ allPosts }) => {
+  const [page, setPage] = useState(1);
```

Comment:
ok, so this is probably acceptable for now, as a baby step, but this is ultimately not how pagination should be done. it's not good for linkability and especially for google.  instead of using state, what we want to do is generate numbered pages, like `/posts/1` and `/posts/2` (sometimes you can do a thing where `/posts` is resolved the same as `/posts/1` for a cleaner url). so we want to avoid using state at all, and just have links to pages of generated paginated posts.  does that make sense?  i'm not opposed to deploying it like this temporarily, but should be on our near-term radar for sure.

Reply:
https://github.com/kiahjh/ancient-path/issues/42 <- todo